### DISCUSSION TITLE
Hindsight With External Power Only

### DISCUSSION THREAD

**Discussion by andrewlongron** [gopro/labs#427] [Feature Requests]
I cannot find this function any where so im assuming that it doesn't exist. 
I am an Avid outdoorsman and Fisherman. Before; we could run the GoPro on Hindsight when it was loop mode with out the battery and strictly off external power. 
This plays a huge function when it comes to overheating as the GoPro would last 2x as long without the battery installed and just on external power. 
Since this function was removed my entire workflow when filming a YT episode has been dead in the water. 
The GoPro wont last more than 20 mins in direct sunlight with Hindsight and the battery installed. 

Hindsight has to be allowed to use external power without the need for a battery in order to get the actual function of the feature for people like me who spend a lot of time outdoors.

**Comment by dnewman-gpsw** [comment]
Hindsight works without a battery here, just tried that on HERO11.

**Reply by andrewlongron** [reply]
Seriously? Mine keeps telling me to insert battery? 
Just to make sure im not missing something I did see one comment that said something along the lines of after the GoPro Open firmware was installed this function was enabled. 
I am just about to install the firmware but wanted to verify this function was possible before I went through the trouble.

**Reply by dnewman-gpsw** [reply]
This is supported without any custom firmware, if you power source provide current maximum metadata (many do not.)  Your power source must report that it can do 2A (over the data lines.)  If the camera doesn't get told the rating for the power source, it will reject it.  Labs FW does have a command to bypass USB power certifications.

**Reply by andrewlongron** [reply]
You know now that you say it, I havnt tried it without the media mod and the media mod was the main root of a few other problems I had. 
I just tried it without the media mod and that worked. I appreciate it!

**Reply by dnewman-gpsw** [reply]
With MediaMod (more power) you can use Labs FW to solve this issue with QR code command !MTUSB=1.
